### DISCUSSION TITLE
Question: Does radicale support the BDAY field from RFC6350?

### DISCUSSION THREAD

**Discussion by SeppPenner** [Kozea/Radicale#1422] [Q&A general] [answered]
See the spec here: https://tools.ietf.org/html/rfc6350#section-6.2.5

**Comment by da4089** [comment]
Radicale uses the `vobject` Python module for parsing vCards.  So support or otherwise is dependent on vObject.

`BDAY` fields are defined for vCard 2.1, 3.0, and 4.0.

vObject does not currently implement any special processing of BDAY fields.  They are parsed, but simply returned as a string.

I'm not sure how Radicale deals with these string values.

**Comment by pbiering** [comment, marked as answer]
I have here a lot of ics files on my server containing several formats:

```
BDAY:YYYY-MM-DD
BDAY:DD.MM.YYYY
BDAY:YYYYMMDD
```
Not seen any issue so far and also not seen any other formats. I would also assume Radicale is not touching this explicitly - there would be no reason for that.

**Reply by SeppPenner** [reply]
Thank you for the information. I once planned to use Radicale instead of Synology Calendar because Synology Calendar has the strange behaviour that it works 99% of the time but sometimes throws strange errors out of nowhere that nobody seems to be able to fix :D
